### PR TITLE
update docker log collection documentation

### PR DESCRIPTION
**Pull request by NBParis**
### What does this PR do?
Update docker log collection instruction for containerized agent
### Motivation

Make sure installation instruction for containerized agent is cristal clear regarding log collection

### CODE REVIEW

File: content/logs/index.md [hunk 1]
```diff
@@ -123,10 +123,14 @@ logs:
 
 Agent 6 is able to collect logs from containers. It can be installed [on the host](https://github.com/DataDog/datadog-agent/blob/master/docs/beta/upgrade.md) or [in a container](https://github.com/DataDog/datadog-agent/tree/master/Dockerfiles/agent).
 
-For containerized installation, mount those two repositories:
-
-* `/var/run/docker.sock`: to make sure the agent can access the Docker API.
-* `/opt/datadog-agent/run`: If the agent container is restarted or removed, the registry needs to know the last logs collected for each monitored container or file.
+For containerized installation, here are the command related to log collection:
+
+* `-v /opt/datadog-agent/run:/opt/datadog-agent/run:rw`: Store on disk where to pick log file or container stdout when we restart
```
Comment (tmichelet) [inline content/logs/index.md]: what happens when this folder doesn't exist on the host? is it created?

File: content/logs/index.md [hunk 2]
```diff
@@ -123,10 +123,14 @@ logs:
...
+* `--net=host`: Give access to the host network
+* `-v /var/run/docker.sock:/var/run/docker.sock:ro`: Give access to docker api to collect container stdout and stderr
+* `-v /my/path/to/conf.d:/conf.d:ro`: mount configuration repository
+* `-v /my/file/to/tail:/tail.log:ro`: Foreach log file that should be tailed by the agent (not required if you only want to collect container stdout or stderr)
+* `-e DD_API_KEY=<YOUR_API_KEY>`: Set the api key
```
Comment (tmichelet) [inline content/logs/index.md]: Does this work? I think we also need `DD_LOG_ENABLED=true`

Comment (NBParis) [reply content/logs/index.md]: @tmichelet Is it already availble in agent beta.5 ?

File: content/logs/index.md (hunk 1 above)
Comment (NBParis) [reply content/logs/index.md]: Good point, will check the behaviour in this case.

File: content/logs/index.md (hunk 2 above)
Comment (tmichelet) [reply content/logs/index.md]: yes

Comment (NBParis) [reply content/logs/index.md]: Nice, will update the doc

File: content/logs/index.md (hunk 1 above)
Comment (NBParis) [reply content/logs/index.md]: It does indeed create the file locally

**Pull request merged**